app/models/submission.rb

```diff
@@ -1,4 +1,10 @@
 class Submission < ApplicationRecord
+  has_one_attached :result
+
   USE_CASES = %w[one two three four].freeze
   validates :use_case, presence: true, inclusion: { in: USE_CASES, message: 'Invalid use case' }
+
+  def as_json(*)
```

Comment:
Is it worth including a test for this function?

Reply:
Good shout.  It's currently tested remotely by the UseCase swagger tests, but explicit model tests are better :+1: